xml/System.Net/ServicePoint.xml

```diff
@@ -317,15 +320,19 @@ Although a <xref:System.Net.ServicePoint> object can make multiple connections t
         <summary>Gets the last client certificate sent to the server.</summary>
         <value>An <see cref="T:System.Security.Cryptography.X509Certificates.X509Certificate" /> object that contains the public values of the last client certificate sent to the server.</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Examples  
- The following code example displays the value of this property.  
-  
+          <format type="text/markdown"><![CDATA[
+## Remarks
+
+> [!NOTE]
+>  Since .NET 9, this property is not implemented.
```

Comment:
This sounds like if there was an implementation in .NET 8, but then we removed it. For unimplemented properties, it would be more accurate to state that

```suggestion
>  This property is only implemented on .NET Framework.
```

Reply:
:facepalm: I did global replace and didn't read it.

Reply:
Fixed.